torch/testing/_internal/distributed/rpc/rpc_test.py

```diff
@@ -240,6 +241,17 @@ def heavy_rpc_torchscript(tensor):
         tensor /= i + 1
     return 0
 
+
+@torch.jit.script
```

Comment:
We should put these tests in to "torch/testing/_internal/distributed/rpc/jit/rpc_test.py".

Reply:
Sure, I'll move it.

I saw both files, but I also saw other torch.jit.script code in the rpc/rpc_test.py file, so I figured that the distinction was fuzzy.